test-series/4/tools/generate-point-sources.ipynb

```diff
@@ -0,0 +1,407 @@
+{
```

Comment:
<p>Maybe this should be hard coded in the above function?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera-validation/pull/78/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>